docs/OMS-MySQL-Instructions.md

```diff
@@ -0,0 +1,88 @@
+# MySQL Server and Log Monitoring Solution for Operations Management Suite
+
+1. Setup a Linux(Ubuntu/Redhat) machine and install [MySQL](http://dev.mysql.com/doc/refman/5.7/en/installing.html).
+
+2. Download and Install [OMS Agent for Linux](https://github.com/Microsoft/OMS-Agent-for-Linux) on the machine. 
+
+3. Configure MySQL to generate slow, error and general logs.
+
+4. Verify and update the MySQL log file path in the configuration file ```/etc/opt/microsoft/omsagent/conf/omsagent.d/mysql.conf```  
+If ```mysql.conf``` is not present in the above location, move it from ```/etc/opt/microsoft/omsagent/sysconf/omsagent.d/```
```

Comment:
Lets add the command for moving it from `/etc/opt/microsoft/omsagent/sysconf/omsagent.d/mysql.conf`

`cp /etc/opt/microsoft/omsagent/sysconf/omsagent.d/mysql.conf /etc/opt/microsoft/omsagent/conf/omsagent.d/mysql.d`